jenkins/agent-maven.yaml

```diff
@@ -13,14 +13,14 @@ spec:
       image: maven:3.8.6-openjdk-8-slim
       env:
       - name: MAVEN_OPTS
-        value: "-Xmx6000m -Xms6000m"
+        value: "-Xmx8000m -Xms8000m"
       resources:
         requests:
-          memory: "8Gi"
-          cpu: "4000m"
+          memory: "10Gi"
+          cpu: "2000m"
```

Comment:
Why is cpu reduced?

Reply:
The goal of this maven plugin integration is to expose components with known vulnerabilities. Check online document here http://jeremylong.github.io/DependencyCheck/.